src/MissionParallelCatchup/parallel_catchup_helm/values.yaml

```diff
@@ -0,0 +1,31 @@
+redis:
+  job_queue: "ranges"
+  success_queue: "succeeded"
+  failed_queue: "failed"
+  progress_queue: "in_progress"
+  request_cpu: "100m"
+  request_memory: "100Mi"
+
+worker:
+  stellar_core_image: "stellar/stellar-core:latest"
+  replicas: 5
+  request_cpu: "1250m"
+  request_memory: "24Gi"
+  request_ephemeral_storage: "35Gi"
+  limit_cpu: "8"
+  limit_memory: "48Gi"
+  limit_ephemeral_storage: "40Gi"
+
+monitor:
+  logging_interval_seconds: 10
+  request_cpu: "100m"
+  request_memory: "100Mi"
```

Comment:
As above it could be nice to switch to the nested `resources` block

Reply:
done